modules/constants.inc.php

```diff
@@ -146,6 +150,41 @@
   define("AUC_BONUS_COW_FOR_VP",      5);
   define("AUC_BONUS_6VP_AND_FOOD_VP", 6);
   define("AUC_BONUS_FOOD_FOR_VP",     7);
+  define("AUC_BONUS_4DEPT_FREE",      8);
+  define("AUC_BONUS_3VP_SELL_FREE",   9);
+
+  // Events
```

Comment:
remove these for now?  I guess more constants isn't the end of the world but worth thinking about.